### PR TITLE
Improve player movement

### PR DESCRIPTION
**Pull request by Alemiz112**

### CODE REVIEW

File: src/main/java/cn/nukkit/Player.java [hunk 1]
```diff
@@ -4627,8 +4627,12 @@ public void sendPosition(Vector3 pos, double yaw, double pitch, int mode, Player
         this.ySize = 0;
 
         if (targets != null) {
+            if (Arrays.stream(targets).anyMatch(target -> target == this)) {
+                this.ySize = 0;
```
Comment (PetteriM1) [inline src/main/java/cn/nukkit/Player.java:4631]: Already set to 0 on line 4627

File: src/main/java/cn/nukkit/Player.java [hunk 2]
```diff
@@ -4627,8 +4627,12 @@ public void sendPosition(Vector3 pos, double yaw, double pitch, int mode, Player
         this.ySize = 0;
 
         if (targets != null) {
+            if (Arrays.stream(targets).anyMatch(target -> target == this)) {
+                this.ySize = 0;
+            }
             Server.broadcastPacket(targets, pk);
         } else {
+            this.ySize = 0;
```
Comment (PetteriM1) [inline src/main/java/cn/nukkit/Player.java:4635]: Already set to 0 on line 4627

File: src/main/java/cn/nukkit/Player.java (hunk 1 above)
Comment (Alemiz112) [reply src/main/java/cn/nukkit/Player.java:4629]: Solved

File: src/main/java/cn/nukkit/Player.java (hunk 2 above)
Comment (Alemiz112) [reply src/main/java/cn/nukkit/Player.java:4633]: Solved

**Pull request closed without merging**